core/src/main/java/com/sequenceiq/cloudbreak/job/nodestatus/NodeStatusCheckerJob.java

```diff
@@ -97,43 +110,87 @@ private void executeNodeStatusCheck(Stack stack) {
             }
             CdpNodeStatusRequest request = requestBuilder.withCmMonitoring(true).build();
             CdpNodeStatuses statuses = nodeStatusService.getNodeStatuses(stack, request);
-            processNodeStatusReport(statuses.getNetworkReport(), NodeStatusProto.NodeStatus::getNetworkDetails, stack, "Network");
-            processNodeStatusReport(statuses.getServicesReport(), NodeStatusProto.NodeStatus::getServicesDetails, stack, "Services");
-            processNodeStatusReport(statuses.getSystemMetricsReport(), NodeStatusProto.NodeStatus::getSystemMetrics, stack, "System metrics");
-            processCmMetricsReport(statuses.getCmMetricsReport(), stack);
+            processNodeStatusReport(statuses.getNetworkReport(), NodeStatusProto.NodeStatus::getNetworkDetails, stack, "Network", accountId);
+            processNodeStatusReport(statuses.getServicesReport(), NodeStatusProto.NodeStatus::getServicesDetails, stack, "Services", accountId);
+            final boolean processMetrics;
```

Comment:
maybe consider `processMetricsToDatabus` here and within `processMnMetricsReport`.

Reply:
technically that is just processing/logging, data is only sent to databus if that is enabled during the processing, im keeping this for now